### PR TITLE
updated the installation instructions, added the pam_install script

### PR DESCRIPTION
**Pull request by vincentberenz**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

Updated the installation documentation. The binary installation was removed, but the installation from source was grandly simplified.

The commit also included the script installation file from source.

## How I Tested

Compiled the html locally

### CODE REVIEW

File: resources/pam_install [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+bashrc=${HOME}/.bashrc
+prefix_dir=${HOME}/.mpi-is/pam
+install_dir=${prefix_dir}/install
+lib_dir=${install_dir}/lib
+bin_dir=${install_dir}/bin
+
+mkdir -p ${install_dir}
+mkdir -p ${lib_dir}
+mkdir -p ${bin_dir}
```
Comment (luator) [inline resources/pam_install:71]: Don't you need to copy the installed files from the workspace to here?  Or is this done by the pam_install package during build?  In this case, there is probably no need for the mkdir as it should already be created?

File: A1_overview_and_installation.md [hunk 2]
```diff
@@ -45,72 +45,30 @@ Installing the software will results in:
...
-> 2. Extract files and subsequently run scripts as explained in the following for installing the necessary dependencies
-> 3. Download and move MuJoCo key to specified location
-> 4. (Optional:) Setup colcon workspace
-> 5. (Optional:) Setup SSH authentification for GitHub
-> 6. Clone project repository from GitHub
-> 7. Compile project with ```colcon```
-> 8. Source terminal with given setup script
-> 9. Run test import commands in terminal
-
 ### From tar archives
```
Comment (luator) [inline A1_overview_and_installation.md]: If I understand correctly, there is no tar archive involved anymore, so the title should be updated?

Comment (vincentberenz) [reply A1_overview_and_installation.md]: +1

File: resources/pam_install (hunk 1 above)
Comment (vincentberenz) [reply resources/pam_install:71]: It is true it is not really necessary, but I see no downside to making extra sure the directories exist.

**Review by luator — APPROVED** [review]

**Pull request merged**